main.go

```diff
@@ -20,8 +20,9 @@ var (
 
 var (
 	// flags
-	action = kingpin.Flag("action", "Any of the following actions: audit.").Envar("ESTAFETTE_EXTENSION_ACTION").String()
-	level  = kingpin.Flag("level", "Level of security you want to check for. It can be: info, low, moderate, high or critical.").Default("low").OverrideDefaultFromEnvar("ESTAFETTE_EXTENSION_LEVEL").String()
+	action   = kingpin.Flag("action", "Any of the following actions: audit.").Envar("ESTAFETTE_EXTENSION_ACTION").String()
+	level    = kingpin.Flag("level", "Level of security you want to check for. It can be: info, low, moderate, high or critical.").Default("low").OverrideDefaultFromEnvar("ESTAFETTE_EXTENSION_LEVEL").String()
```

Comment:
If you run `npm audit` without the severity level flag does it fail on `info`? If that's the case we should probably default to info level.

Reply:
The default is `low`: https://github.com/npm/cli/blob/latest/lib/config/defaults.js#L113

Reply:
I just noticed actually there's no `info` level in the latest version. Not sure where did I get that. I will remove this option.